### PR TITLE
Ira is creating pull request.

### PR DESCRIPTION
**Pull request by icaes8**
Ira is creating pull request description

### CODE REVIEW

**Comment by icaes8** [comment]
ira is fix for problem

File: _slides/11-icaes8.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Ira is here!
```
Comment (jasontiedt) [inline _slides/11-icaes8.md:6]: LGTM

File: _slides/11-icaes8.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-![droidtocat](https://octodex.github.com/images/droidtocat.png)
+![droidtocat](https://octodex.github.com/images/droidtocat.png)
```
Comment (jasontiedt) [inline _slides/11-icaes8.md:8]: This URL did not change, need to revert

**Review by jasontiedt — APPROVED** [review]
LGTM

File: _slides/11-icaes8.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (Bran8444) [inline _slides/11-icaes8.md:4]: Message

**Review by Bran8444 — APPROVED** [review]
Complete

**Pull request merged**